jsonschema-core.xml

```diff
@@ -123,63 +123,109 @@
                 data in several ways.
             </t>
 
-            <section title="Assertions">
+            <section title="Keyword Behaviors">
                 <t>
-                    JSON Schema can be used to assert constraints on a JSON document, which
-                    either passes or fails the assertions.  This approach can be used to validate
-                    conformance with the constraints, or document what is needed to satisfy them.
+                    JSON Schema keywords fall into several general behavior categories.
+                    Assertions validate that an instance satisfies constraints, annotations
+                    attach information that applications may use in any way they see fit,
+                    and applicators allow for building more complex schemas than a single
```

Comment:
Would it be accurate to say something like "Applicators allow a sub-schema to be applied recursively to some part of the instance, or the entire instance." ?
The current description wasn't clear to me.

Reply:
Throughout these PRs (and earlier changes) I classify applicators as "in-place" or "child" applicators.  I would not say "the entire instance" because people will take that to mean the whole instance document, which may be a much larger chunk of JSON than the current location.

Just saying "Applicators allow a sub-schema to be applied to some part of the instance." might be fine, though?  We'd just leave the enumeration of the different locations to the more detailed explanation.